base/Dockerfile

```diff
@@ -1,35 +1,5 @@
-FROM alpine:3.16.2 as builder
+FROM python:3.10.5-alpine3.16
```

Comment:
Are we able to specify the exact alpine version as 3.16.2?

Reply:
Unfortunately we're at the mercy for the python guys for their latest versions afaik.

I haven't checked what the latest base package is for them.

Reply:
However, latest version of python alpine is running latest version of Alpine.

```
docker run -it --name alpine --rm -v "$(pwd)":/data 'python:3.10.6-alpine3.16' bin/sh
/ # cat /etc/os-release
NAME="Alpine Linux"
ID=alpine
VERSION_ID=3.16.2
PRETTY_NAME="Alpine Linux v3.16"
HOME_URL="https://alpinelinux.org/"
BUG_REPORT_URL="https://gitlab.alpinelinux.org/alpine/aports/-/issues"
```